src/scripts/find_dadis_transboundary_ids.py

```diff
@@ -0,0 +1,221 @@
+import argparse
+import csv
+from typing import TextIO
+
+import pandas as pd
+
+from dadis_client import DadisClient
+
+
+def full_matching_workflow(
+    input_filename: str, output_filename: str, dadis_api_key: str
+) -> pd.DataFrame:
+    """
+    Perform the full matching workflow:
+
+    - Read VBO data from input_filename
+    - Match to DADIS to get dadis_transboundary_id
+    - Save to a new TSV file at output_filename
+    """
+    client = DadisClient(api_key=dadis_api_key)
+    vbo_data = read_vbo_data(input_filename)
+    matched_breeds = match_vbo_breeds(vbo_data=vbo_data, client=client)
+
+    print(f"Writing output file to {output_filename}:")
+    output_file = create_output_tsv(
+        input_filename=input_filename,
+        output_filename=output_filename,
+        extra_cols=["dadis_transboundary_id"],
+    )
+    matched_breeds.to_csv(output_file, sep="\t", index=False, header=False)
+    output_file.close()
+    print("Output written.")
```

Comment:
ususally we preferr logging.info or logging.warn to be able to control the chattyness of our pipeline scripts. No bother though.

Reply:
I usually do the same for anything larger than a script, I've updated the code to use `logging`

Reply:
great!